app/utils/Navigator.scala

```diff
@@ -255,6 +256,11 @@ class Navigator @Inject()() extends Logging with FeatureSwitching {
       mandatoryTrue = MandatoryInformationId,
```

Comment:
remove this old nav entry for exemption page, it's not used now that you've added the nextOn for VATExemptionId